qemu/tests/x86_cpu_flags.py

```diff
@@ -36,7 +36,14 @@ def run(test, params, env):
     error_context.context("Try to log into guest", logging.info)
     session = vm.wait_for_login()
     if params["os_type"] == "linux":
+        check_guest_cmd = params.get("check_guest_cmd")
         check_cpu_flags(params, flags, test, session)
+        if check_guest_cmd:
+            expect_items = params.get("expect_items")
+            if expect_items:
+                results = session.cmd_output(check_guest_cmd % expect_items)
```

Comment:
Prefer to use session.cmd_status. 
```
# echo "124sdfsd" > /tmp/aa
# grep 124 /tmp/aa
124sdfsd
# echo $?
0
# grep 123 /tmp/aa
# echo $?
1
```

Reply:
> cmd_status

Thanks and update.

Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.intel.arch_capabilities: PASS (193.53 s)